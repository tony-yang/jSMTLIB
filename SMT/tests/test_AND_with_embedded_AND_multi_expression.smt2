(set-logic QF_LRA)
(set-info :source | SMT-COMP'06 organizers |)
(set-info :smt-lib-version 2.0)
(set-info :category "check")
(set-info :status sat)
(set-info :notes |This benchmark is designed to check if the DP supports bignumbers.|)
(declare-fun x1 () Real)
(declare-fun x2 () Real)
(declare-fun x3 () Real)
(declare-fun x4 () Real)
(declare-fun x5 () Real)
(declare-fun x6 () Real)
(declare-fun x7 () Real)
(assert (and (and (and (>= (- x1 x2) 86) (>= (- x2 x3) 99)) (>= (- x4 x1) 92)) (and (and (>= (- x5 x6) 94) (>= (- x5 x7) 98)) (>= (- x4 x5) 63))))
(check-sat)
(exit)
