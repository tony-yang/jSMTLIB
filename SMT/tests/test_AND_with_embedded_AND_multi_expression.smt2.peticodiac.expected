# set_logic: QF_LRA
# :source: | SMT-COMP'06 organizers |
# :smt-lib-version: 2.0
# :category: "check"
# :status: sat
# :notes: |This benchmark is designed to check if the DP supports bignumbers.|
p cnf 7 6
c 1.0 -1.0 0.0 0.0 0.0 0.0 0.0
b 7 >=:86.0 NO_BOUND
c 0.0 1.0 -1.0 0.0 0.0 0.0 0.0
b 8 >=:99.0 NO_BOUND
c -1.0 0.0 0.0 1.0 0.0 0.0 0.0
b 9 >=:92.0 NO_BOUND
c 0.0 0.0 0.0 0.0 1.0 -1.0 0.0
b 10 >=:94.0 NO_BOUND
c 0.0 0.0 0.0 0.0 1.0 0.0 -1.0
b 11 >=:98.0 NO_BOUND
c 0.0 0.0 0.0 1.0 -1.0 0.0 0.0
b 12 >=:63.0 NO_BOUND
